<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskDialog</class>
 <widget class="QDialog" name="AddTaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加任务</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/add.ico</normaloff>:/image/add.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="urlTextEdit">
       <property name="toolTip">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="documentTitle">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入将要下载的文件地址，以分号（英文）或换行区分。</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>下载路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pathLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pathChooseButton">
         <property name="text">
          <string>选择下载路径</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="startDownloadButton">
       <property name="text">
        <string>开始下载</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
